_posts/2022-05-27-gsoc-students.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Google Summer of Code Students 2022
+---
+
+We are happy to announce that MDAnalysis is hosting two [GSoC][gsoc]
+students this year -- @aya9aladdin, and @BFedder. MDAnalysis has been accepted as its own
+[organization with GSoC][mda-gsoc] for the third year running and we are grateful to Google for granting us
+the opportunity to undertake _two very exciting GSoC projects_!
+
+## Aya Mohamed Alaa: Context-aware guessers
+
+<img
+src="Aya's Picture Here"
+title="Aya Mohamed Alaa" alt="Aya Mohamed Alaa"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Write something about your project and yourself here (don't forget to link to the project page on the GSoC website)!
+
+Add where we can find you here! (github, twitter, others)
+
+Add a link to your blog here!
+
+
+## Bjarne Feddersen: Adding Energy Readers to MDAnalysis
+
+<img
+src="Bjarne's Picture Here"
+title="Bjarne Feddersen" alt="Bjarne Feddersen"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Write something about your project and yourself here (don't forget to link to the project page on the GSoC website)!
+
+Add where we can find you here! (github, twitter, others)
+
+Add a link to your blog here!
+
+
+— @jbarnoud @hmacdope @ojeda-e @IAlibay @fiona-naughton @orbeckst @lilyminium (mentors)
```

Comment:
I hope I haven't forgotten someone we have so many mentors 🙈 

@richardjgowers and @PicoCentauri you weren't on the mentor list for either students, would you still like to be involved as a mentor? (we can still add you as an extra mentor on either projects or you can be "unofficial" mentors if you'd like).